run.py

```diff
@@ -253,6 +261,43 @@ def internal_server_error_handler(e):
         layout = LayoutHelper(logged_in=False)
     return render_template('500.html', layout=layout.get_entries()), 500
 
+@app.errorhandler(503)
+def service_unavailable_handler(e):
+    if g.user is not None:
+        details = DetailsForTemplate(g.user.id, admin_dashboard=False)
+        layout = LayoutHelper(logged_in=True, details=details)
+    else:
+        layout = LayoutHelper(logged_in=False)
+    return render_template('503.html', layout=layout.get_entries()), 503
+
 if __name__ == '__main__':
     log.debug("Running App.")
     app.run(host='0.0.0.0')
+
+@app.route('/401')
```

Comment:
Please remove all these routes, I recommended it to you for testing in your local set-up. Also, looks like all of your routes have same function name, that can not be correct?